formidable/locale/fr/LC_MESSAGES/django.po

```diff
@@ -28,23 +28,23 @@ msgstr "Valeur vide refusée pour {attr} dans {name}"
 
 #: forms/validations/presets.py:169
 msgid "Missing presets arguments : {}"
-msgstr ""
+msgstr "Paramètre(s) de règle manquant : {}"
 
 #: forms/validations/presets.py:175
 msgid "Extra presets arguments : {}"
-msgstr ""
+msgstr "Excès de paramètres de règle"
```

Comment:
ERROR! we can't keep this crowdin string, it'll break, since the ``{}`` is missing